content/ru/docs/contribute/participating.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Участие в SIG Docs
+content_template: templates/concept
+card:
+  name: contribute
+  weight: 40
+---
+
+{{% capture overview %}}
+
+SIG Docs — это одна из [специальных групп](https://github.com/kubernetes/community/blob/master/sig-list.md) в проекте Kubernetes, которая занимается написанием, обновлением и поддержкой документации Kubernetes в целом. Перейдите на страницу про [SIG Docs в GitHub-репозитории](https://github.com/kubernetes/community/tree/master/sig-docs), чтобы узнать подробную информацию об этой группе.
+
+SIG Docs активно принимает правки и дополнения в документацию, так и отзывы от всех участников. Любой может открыть пулреквест (PR), либо сообщить про ошибки в тексте или просто прокомментировать выполняемые пулреквесты.
+
+Вы также можете стать [членом](#members), [рецензентом](#reviewers) или [утверждающим](#approvers). Эти роли расширяют ваши возможности, но и предлагают выполнение определенных обязанностей по рассмотрению и принятию изменений. Изучите содержимого файла [community-membership](https://github.com/kubernetes/community/blob/master/community-membership.md) в директории сообщества репозитория, чтобы узнать про членство в сообществе Kubernetes. В остальной части этой страницы кратко рассматривается функционирование ролей в группе SIG Docs, которая в совокупности отвечает за поддержание одного из самой публичной части Kubernetes — сайта и документации Kubernetes.
```

Comment:
@lex111: The links to `#members`, `#reviewers` and `#approvers` do not match the localized titles of those sections.